packages/dev/core/src/Materials/shaderMaterial.ts

```diff
@@ -1022,7 +1028,9 @@ export class ShaderMaterial extends PushMaterial {
             }
         }
 
-        const mustRebind = mesh && storeEffectOnSubMeshes ? this._mustRebind(scene, effect, subMesh, mesh.visibility) : scene.getCachedMaterial() !== this;
+        const mustRebind = true; //mesh && storeEffectOnSubMeshes ? this._mustRebind(scene, effect, subMesh, mesh.visibility) : scene.getCachedMaterial() !== this;

```

Comment:
Is this meant to be commented out? If so, could `mustRebind` be removed? It ends up being `&& true` in the conditionals where it's used below.

Reply:
That has been fixed since then:)